### ISSUE TITLE
ML-Agent 1.7.0 doesn't specify dependence on package Unity Analytics

### ISSUE THREAD

**Issue by Faulo** [Unity-Technologies/ml-agents#4790]
**Describe the bug**
ML-Agent 1.7.0 references classes in the [UnityEngine.UnityAnalyticsModule](https://docs.unity3d.com/ScriptReference/UnityEngine.UnityAnalyticsModule.html) namespace, but doesn't say so in its [package.json](../blob/master/com.unity.ml-agents/package.json). Therefore, if the Unity Analytics module is disabled for whatever reason, ML-Agents can't compile.

**To Reproduce**
1. Add ML-Agents 1.7.0 to a Unity project
2. Disable Unity's built-in package ```com.unity.modules.unityanalytics```
3. Receive attached compilation errors

**Console logs**
```
Library\PackageCache\com.unity.ml-agents@1.7.0-preview\Runtime\Analytics\InferenceAnalytics.cs(51,13): error CS1069: The type name 'AnalyticsResult' could not be found in the namespace 'UnityEngine.Analytics'. This type has been forwarded to assembly 'UnityEngine.UnityAnalyticsModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' Enable the built in package 'Unity Analytics' in the Package Manager window to fix this error.
Library\PackageCache\com.unity.ml-agents@1.7.0-preview\Runtime\Analytics\InferenceAnalytics.cs(51,38): error CS0012: The type 'AnalyticsResult' is defined in an assembly that is not referenced. You must add a reference to assembly 'UnityEngine.UnityAnalyticsModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
Library\PackageCache\com.unity.ml-agents@1.7.0-preview\Runtime\Analytics\InferenceAnalytics.cs(55,27): error CS0103: The name 'AnalyticsResult' does not exist in the current context
Library\PackageCache\com.unity.ml-agents@1.7.0-preview\Runtime\Analytics\InferenceAnalytics.cs(116,13): error CS0012: The type 'AnalyticsResult' is defined in an assembly that is not referenced. You must add a reference to assembly 'UnityEngine.UnityAnalyticsModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
```

**Screenshots**
![grafik](https://user-images.githubusercontent.com/7897059/102897007-a118a800-4467-11eb-9d8c-8455fea38aa2.png)

**Environment:**
- Unity Version: Unity 2019.4.14f1
- OS + version: Windows 10
- _ML-Agents version_: ML-Agents v1.7.0

**Workaround**
Until this is fixed, you can just add the line ```"com.unity.modules.unityanalytics": "1.0.0"``` to your project's manifest.json (or enable it via the Package Manager, as the error states).

**Comment by surfnerd** [comment]
Hi @Faulo,
Thanks for finding this.  We will fix it ASAP

**Comment by surfnerd** [comment]
Hi @Faulo,
The fix was merged in #4794 and will be released as a hotfix.
